script.js

```diff
@@ -0,0 +1,76 @@
+function startGame() {
+
+  for(var i = 1; i <= 9; i = i + 1) {
+    clearBox(i);
+  }
+
+  document.turn = "X";
```

Comment:
While adding these properties to `document` works,  would recommend using something specific to your problem domain (e.g. `tictactoe.turn`).